src/components/MediaItem/MediaItemTitle.tsx

```diff
@@ -12,6 +12,8 @@ type Props = {
   parentRef: RefObject<HTMLDivElement>
 }
 
+const top3Colors = ['text-yellow-300', 'text-gray-300', 'text-yellow-700']
```

Comment:
Maybe change to just `topColors` instead, that way if we decide to increase the amount in the future, wont need to rename it.